guide/inflation.rst

```diff
@@ -83,21 +83,32 @@ DART includes advanced adaptive inflation algorithms that allow inflation
 to vary dynamically in both space and time. These algorithms treat inflation
 as a random variable characterized by a probability density function (PDF).
 The mean (or mode) of the PDF is used as the inflation value, while the standard
-deviation reflects the level of confidence in that value.
+deviation reflects the level of confidence in that value. In this way, the 
+inflation parameters (mean, standard deviation) are prognostic variables and
```

Comment:
I am going to omit the term 'prognostic variables'. This is typically used to describe a variable that the model predicts through integrating a physical equation. To me, a prognostic variable is a variable that has a partial derivative with respect to time in the model PDEs. Inflation mean and standard deviation is like a physical model parameter. I'll reword and push the change.